SportHub/src/modules/SearchResultsList/components/SearchResultsList.js

```diff
@@ -94,35 +96,37 @@ const SearchResultsList = ({ contentSearchValue }) => {
         loader={<p>{t('LoadingText')}</p>}
       >
         {contentSearchValue && articles.map((article) => (
-          <div key={article.id} onClick={() => navigate(ROUTES.LOGIN)} className={styles.option}>
-            <ul>
-              <li>
-                <span>{article.category.categoryName}</span>
-                {
-                  article.subCategory
-                    ?
-                    <>
-                      <span>{" > "}</span>
-                      <span>{article.subCategory.subCategoryName}</span>
-                    </>
-                    :
-                    <></>
-                }
-                {
-                  article.team
-                    ?
-                    <>
-                      <span>{" > "}</span>
-                      <span>{article.team.teamName}</span>
-                    </>
-                    :
-                    <></>
-                }
-                <span>{" > "}</span>
-                <span>{article.title}</span>
-              </li>
-              <li><span>{getMatchedMainText(article.mainText)}</span></li>
-            </ul>
+          <div className={styles.container}>
+            <div key={article.id} onClick={() => navigate(ROUTES.LOGIN)} className={styles.option}>
```

Comment:
Is this correct route?

Reply:
There is no route for reading an article yet